conda-forge.yml

```diff
@@ -1,8 +1,9 @@
+bot:
+  automerge: true
+  inspection: update-grayskull
```

Comment:
I think basically any package that used `automerge: true` should also have `inspection: update-grayskull`.  Otherwise, upstream dependency changes go unnoticed and things break downstream.

Reply:
I fully agree - I just did not get around to update all the recipes yet.